carbonmark/components/Layout/AddressSection/index.tsx

```diff
@@ -1,32 +1,50 @@
-import { CopyAddressButton } from "@klimadao/lib/components";
+import { Anchor, CopyAddressButton } from "@klimadao/lib/components";
 import { Domain } from "@klimadao/lib/types/domains";
-import { concatAddress } from "@klimadao/lib/utils";
+import { WalletProvider, concatAddress } from "@klimadao/lib/utils";
 import { Trans } from "@lingui/macro";
+import LaunchIcon from "@mui/icons-material/Launch";
 import { Text } from "components/Text";
-import { FC } from "react";
+import { FC, useEffect, useState } from "react";
 import * as styles from "./styles";
 
 interface AddressSectionProps {
   domain?: Domain;
   address?: string;
 }
 export const AddressSection: FC<AddressSectionProps> = (props) => {
+  const [isTorusWallet, setIsTorusWallet] = useState(false)
+  useEffect(() => {
+    setIsTorusWallet(localStorage.getItem("web3-wallet") == WalletProvider.TORUS);
+  }, [])
+
   return (
     <div className={styles.address}>
       <Text t="body1">
         <Trans>Your Wallet Address:</Trans>
       </Text>
-      {!props.address && (
+      {false && (
         <Text t="body1" color="lightest" uppercase>
           <Trans>Not Connected</Trans>
         </Text>
       )}
-      {props.address && (
-        <CopyAddressButton
-          label={concatAddress(props.address)}
-          address={props.address}
-          className="copyButton"
-        />
+      {true /* props.address */ && (
+        <div>
+          <CopyAddressButton
+            label={concatAddress(props.address || "0x123456789")}
+            address={props.address}
+            className="copyButton"
+          />
+          {true /* isTorusWallet */ && (
+            <Anchor
+              className={styles.iconAndText}
+              href={`https://polygon.tor.us/`}
+            >
+              <Text className={styles.externalLink} t="body2" uppercase>
+                View <LaunchIcon />
```

Comment:
hey @ladytrekker, what are the steps required to prepare this string for lingui?

Reply:
You just need to wrap the actual string into Trans or t

So like

```TS
  <Text className={styles.externalLink} t="body2" uppercase>
    <Trans>View</Trans>{" "}<LaunchIcon />
   </Text>
```